keda/Chart.yaml

```diff
@@ -8,7 +8,7 @@ kubeVersion: ">=v1.23.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.17.2
+version: 2.17.3
```

Comment:
please don't bump the version, this will be done as part of the release workflow